bigquery_etl/util/common.py

```diff
@@ -53,9 +53,15 @@ def random_str(length: int = 12) -> str:
     return "".join(random.choice(string.ascii_lowercase) for i in range(length))
 
 
-def render(sql_filename, format=True, template_folder="glean_usage", **kwargs) -> str:
+def render(
+    sql_filename,
+    format=True,
+    template_folder="glean_usage",
```

Comment:
Since this is now being more generally used, IMO it'd be better to not have this default to `glean_usage`, but instead explicitly specify `template_folder="glean_usage"` where that's applicable.